public/v4/apps/radarr.yml

```diff
@@ -0,0 +1,66 @@
+captainVersion: 4
+services:
+    # Radarr
+    $$cap_appname:
+        image: linuxserver/radarr:$$cap_radarr_version
+        environment:
+            TZ: $$cap_tz
+            PUID: $$cap_puid
+            PGID: $$cap_guid
+        volumes:
+            - $$cap_appname-config:/config
+            - $$cap_path_to_movies:/movies
+            - $$cap_path_to_downloads:/downloads
+        caproverExtra:
+            containerHttpPort: 7878
+caproverOneClickApp:
+    variables:
+        - id: $$cap_radarr_version
+          label: Radarr Version
+          defaultValue: amd64-latest
```

Comment:
Let's change this to a fixed version.

Also, looks like they have multiarch tags as well. Let's use those:
![image](https://user-images.githubusercontent.com/10582082/103322292-97fd7d00-4a0b-11eb-968d-39e9f80684e5.png)

Reply:
those multi arch is equivalent to [version-3.0.1.4259](https://hub.docker.com/layers/linuxserver/radarr/version-3.0.1.4259/images/sha256-9e65bfed06f9cadeef2b04ec5fd42f08316c03858ade90e2eb407d3176a106df?context=explore)
should i change it to that ?

Reply:
Yea - that looks good

Reply:
Oki

Reply:
FYI - since you're contributing a lot to these one click apps it's good for you to know the rationale behind these guidelines, e.g., not using `latest` tag. The reason behind that is that a new version might have a different directory mapping, different env vars and etc. Using a specific tag ensures that the one click app will continue to work forever.

Reply:
Yea i forgot about that